qiskit_experiments/library/calibration/fine_amplitude.py

```diff
@@ -21,14 +21,20 @@
 from qiskit.providers import Backend
 from qiskit.pulse.schedule import ScheduleBlock
 
-from qiskit_experiments.framework import BaseExperiment, Options
+from qiskit_experiments.framework import Options
 from qiskit_experiments.library.calibration.analysis.fine_amplitude_analysis import (
     FineAmplitudeAnalysis,
 )
 from qiskit_experiments.exceptions import CalibrationError
+from qiskit_experiments.framework.experiment_data import ExperimentData
+from qiskit_experiments.calibration_management.update_library import Amplitude
+from qiskit_experiments.calibration_management.calibrations import Calibrations
```

Comment:
What happens if we import them to `calibration_management` namespace? Do we get cyclic import error?

Reply:
We always seems to import from the files which is fine.